desafio-6/desafio-6.css

```diff
@@ -0,0 +1,53 @@
+body{
+  font-family: arial, sans-serif;
+  background: #8FBC8F;
+  margin: 0;
+}
+p{
+  padding: 20px;
+  text-align: justify;
+}
+
+.container{
+  height: 60px;
+  background-position: right center;
+  background-size: cover;
+  border-bottom: 1px solid #ffffff;
+}
+
+.container-info{
+  text-align: center;
+  color: white;
+  margin-top: 30px;
+}
+
+.container-info h1{
+  text-transform: uppercase;
+  font-size: 30px;
+  text-shadow: 3px 3px 2px #333;
+}
+
+.container nav ul{
+  text-align: right;
```

Comment:
you could take   of using display flex to do the alignment instead of using text-align: right; 
with display flex you can use `justify-content` and align the whole block.

The text-align property specifies the horizontal alignment of text in an element.
In that case, you need to align your container, not the text.

Reply:
true, forgetting the justify-content property